src/Microsoft.ML.Transforms/GcnTransform.cs

```diff
@@ -444,8 +444,9 @@ protected override Delegate GetGetterCore(IChannel ch, IRow input, int iinfo, ou
                             (ref VBuffer<Float> dst) =>
                             {
                                 getSrc(ref src);
-                                Float mean = Mean(src.Values, src.Count, src.Length);
-                                Float divisor = StdDev(src.Values, src.Count, src.Length, mean);
+                                var srcValues = src.GetValues();
+                                Float mean = Mean(srcValues, src.Length);
```

Comment:
>Float [](start = 32, length = 5)

Hi @eerhardt, I notice here and elsewhere you've been trying to maintain this `Float` stuff. This was an artifact of something roughly four years ago where we tried to maintain separate "double" and "single" builds, and we've been opportunistically removing them, since the desired final state is we have none of this at all.

By this I mean you should choose whichever is less effort for you. If you find it's causing you effort to keep doing this `Float` stuff then please don't bother, it is intended to be removed in due time anyway.

Reply:
Makes sense, probably makes merges/rebases easier.